Search Console redirects

samglover opened the pull request:
This PR addresses over 2,300 404s resulting from the new Court Forms Online, as reported by Google Search Console. Apparently the old website included PDF templates in the `/forms/` and `/pdfs/` directory. This PR will redirect those URLs to `/forms` (only for URLs that include `.pdf`).

It also includes a regex for URLs containing `/slack_archive/`, and redirects them to the documentation website.

And there are a few specific (non-regex) redirects.

I also removed the line that disabled image optimization, as optimized images seem like a good thing. It works fine locally, so it seems worth trying.

nonprofittechy reviewed (changes requested):
I think we should check in with @colarusso on this--most of the broken links seem to be from the form explorer page, and maybe we want to restore that content.

samglover commented:
Since the plan is to move Form Explorer to its own subdomain (either projects.suffolklitlab.org/formexplorer or formexplorer.suffolklitlab.org), I'll hold off and update this PR when that has happened.

samglover commented:
I've updated the redirects so that Form Explorer PDF files like `https://courtformsonline.org/forms/6b4ebd487f82f387512ac20da28803db.pdf` will redirect to `https://formexplorer.suffolklitlab.org/forms/6b4ebd487f82f387512ac20da28803db.pdf`. I've also updated the redirect for the Form Explorer CSV.

I removed the other redirects. I'd like to wait a couple more weeks for Google's search index to "settle" before I redirect specific URLs. I've already seen some 404s disappear as Google updates its index, and there's no need to clutter the config file with unnecessary redirects.
